src/Microsoft.DotNet.Cli.Utils/CommandResolver.cs

```diff
@@ -69,7 +70,7 @@ private static CommandSpec ResolveFromRootedCommand(string commandName, IEnumera
 
             if (commandPackage == null) return null;
 
-            var depsPath = projectContext.GetOutputPathCalculator().GetDepsPath(Constants.DefaultConfiguration);
+            var depsPath = projectContext.GetOutputPathCalculator(Constants.DefaultConfiguration).RuntimeFiles.Deps;
```

Comment:
This is really strange, the class is called OutputPathCalculator and you are putting these properties on it. If you want the properties, I would create a new Entity that the Calculator can return, like an Output or something like that.